Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/questions.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import QuestionField from "./question";
+import Button from "@material-ui/core/Button";
+import "./questions.css";
+
+class QuestionFields extends Component {
```

Comment:
why the class component here vs functional? Nothing wrong with it, up to the team though if you want to keep everything consistent

Reply:
Yes working on switching them! Thank you should have it done by tomorrow!